website/content/en/docs/overview/cheat-sheet.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Cheat Sheet"
+linkTitle: "Cheat Sheet"
+weight: 3
+description: >
+    Operator-SDK Cheat Sheet commands and operations
+---
+
+Below you will find a cheat sheet with options and helpers for projects, which are built with the SDK and are respecting its proposed layout.
+
+## Base commands and options 
+
+| Command   | Description  |
+|-------|-----------|
+| `operator-sdk init`          | To initialize an operator project in the current directory |
+| `operator-sdk init --plugins=<plugin-key>`          | To initialize an operator project in the current directory using a specific plugin. You can check the available plugins you can run `oparator-sdk --help`. E.g (`operator-sdk init --plugins=helm`)|
+| `operator-sdk create api [flags]`          | Let you create your own APIs with its [GKV][gkvs] by [Extendig the Kubernetes API with CustomResourceDefinitions][extend-k8s-api], or let you use external or core-types as well as generate their respective [controllers][controllers-k8s-doc]|
```

Comment:
```suggestion
| `operator-sdk create api [flags]`          | Lets you create your own APIs with its [GKV][gkvs] by [Extending the Kubernetes API with CustomResourceDefinitions][extend-k8s-api], or lets you use external/core-types. Also generates their respective [controllers][controllers-k8s-doc]
```

Reply:
Do you think it'll be useful to have links to the commands pointing to the explanation of the command in https://sdk.operatorframework.io/docs/cli/. 
For ex: 
[operator-sdk create api](https://sdk.operatorframework.io/docs/cli/operator-sdk_create_api/) | Lets you create.....etc

Reply:
I think that could be nice. However, a little hard to keep maintained. However, see that at the end I added 

> **NOTE:** For further information check about the `make` options see the Makefile scaffolded in your project. Also, see the 
[CLI][cli] documentation to check all possible options.

Would that be fine?

Reply:
I am closing this one since the change was addressed. However, please feel free to ping me if you see that would be better we add the links for the cli commands individually.

Reply:
No worries, this is fine..

Reply:
I disagree. If the link needs updating, then so does the command here, since the link's resource maps exactly to the command names.

Reply:
Once again, I really don't think command information that can be gotten from `operator-sdk --plugins=<plugin> --help` should be in the cheatsheet. Autogenerated CLI docs are already quite informative.